src/Symfony/Component/Form/ButtonBuilder.php

```diff
@@ -709,27 +710,31 @@ public function getDataLocked()
      */
     public function getFormFactory()
     {
+        return null;
```

Comment:
This breaks the interface contract. Even if someone should not call this for a button.

Should throw an exception instead?

Reply:
Makes sense to me 👍